sources/install/package_cloud.sh

```diff
@@ -137,6 +137,11 @@ function install_azure_cli() {
     curl -sL https://aka.ms/InstallAzureCLIDeb -o /tmp/azure-cli-install.sh
     bash /tmp/azure-cli-install.sh
     rm /tmp/azure-cli-install.sh
+
+    # Move az from /opt directory
+    mv /opt/az /opt/tools/az
+    mv /usr/bin/az /opt/tools/bin/
+    sed -i 's|"$bin_dir"/../../opt/az/bin/python3|"$bin_dir"/../az/bin/python3|g' /opt/tools/bin/az
```

Comment:
Replace with full path instead, we dont need the dynamically resolved pwd path:
`sed -i 's|"$bin_dir"/../../opt/az/bin/python3|/opt/tools/az/bin/python3|g' /opt/tools/bin/az`